src/Raven.Server/Documents/Handlers/BatchRequestParser.cs

```diff
@@ -741,7 +741,7 @@ private static void ThrowUnexpectedToken(JsonParserToken jsonParserToken, JsonPa
         {
             // Although we using here WithCancellation and passing the token,
             // the stream will stay open even after the cancellation until the entire server will be disposed.
-            var read = await stream.ReadAsync(buffer.Buffer.Array, buffer.Buffer.Offset, buffer.Buffer.Count, token).WithCancellation(token);
+            var read = await stream.ReadAsync(buffer.Buffer.Array, buffer.Buffer.Offset, buffer.Buffer.Count, token);
```

Comment:
the above comment is no longer valid - we don't use WithCancellation anymore. why did we need to add it here as we pass `token` anyway